### PR TITLE
Rename and move python tests and conftest.py to a proper dir

### PR DESCRIPTION
**Pull request by volpatto**
Hi all,

This PR aims to solve #144.

The modifications are the following:

* Several Python tests were scattered in Cpp source directories. I moved them to `tests` dir, which I think that is the correct dir.

* Old `conftest.py` inside Cpp source dir was merged with `conftest.py` inside `tests`. All fixtures are preserved then.

Cheers!

### CODE REVIEW

**Comment by volpatto** [comment]
Well... this is failing due to some `ThermoFun` issue. I don't know how to solve it, but I noticed that it has a recent new release. I don't know it it's related though.

**Comment by volpatto** [comment]
> Well... this is failing due to some `ThermoFun` issue. I don't know how to solve it, but I noticed that it has a recent new release. I don't know if it's related though.

Pinning `ThermoFun` does not solve the problem, unfortunately

**Comment by tadeu** [comment]
@volpatto, I'll check the `ThermoFun` issue

**Comment by volpatto** [comment]
> @volpatto, I'll check the `ThermoFun` issue in a different PR

Thanks! Fixed in #148

File: tests/conftest.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 
 import pytest
+import numpy as np
+
+from reaktoro import ChemicalEditor, ChemicalSystem, Partition
```
Comment (tadeu) [inline tests/conftest.py:6]: It seems some of these are already imported in line 9, this line could be removed and `Partition` could be added there

Comment (volpatto) [reply tests/conftest.py:6]: Ops! my bad

**Pull request merged**